Compiling Error

I had this error:

Linking TestUnitTest++...
Running unit tests...
terminate called after throwing an instance of 'char const_'
Makefile:79: recipe for target 'TestUnitTest++' failed
make[2]: *_\* [TestUnitTest++] Aborted (core dumped)
make[2]: **\* Deleting file 'TestUnitTest++'
make[2]: Leaving directory '/tmp/tmp/QuickFix/quickfix/UnitTest++'
Makefile:495: recipe for target 'all-recursive' failed
make[1]: **\* [all-recursive] Error 1
make[1]: Leaving directory '/tmp/tmp/QuickFix/quickfix'
Makefile:404: recipe for target 'all' failed
make: **\* [all] Error 2

Please, check: http://paste.debian.net/152819/

--- a/UnitTest++/src/TestMacros.h
+++ b/UnitTest++/src/TestMacros.h
@@ -109,5 +109,10 @@
 
 #define TEST_FIXTURE(Fixture,Name) TEST_FIXTURE_EX(Fixture, Name, UnitTest::Test::GetTestList())
 
+#if __cplusplus >= 201103L
+#define NOEXCEPT_FALSE noexcept(false)
+#else
+#define NOEXCEPT_FALSE
+#endif
 
 #endif

--- a/UnitTest++/src/tests/TestTestMacros.cpp
+++ b/UnitTest++/src/tests/TestTestMacros.cpp
@@ -131,7 +131,7 @@ TEST(FixturesWithThrowingCtorsAreFailures)
 
 struct FixtureDtorThrows
 {
-	~FixtureDtorThrows() { throw "exception"; }
+	~FixtureDtorThrows() NOEXCEPT_FALSE { throw "exception"; }
 };
 
 TestList throwingFixtureTestList2;

